### PR TITLE
docs(themes): Adding jp files

### LINKED ISSUE #1501: Create article/section that demonstrates how to use the new @media print functionality
`igx-core()` now includes styles for printing layout. We should consider adding a topic that explains the benefits that it gives and how to handle scenarios like `how to handle column hiding when printing`.

Related issue https://github.com/IgniteUI/igniteui-angular/issues/1995

### PR DESCRIPTION
**Pull request by desig9stein**
I think to create a sample that demonstrates the printing capability of the Grid before merging the documentation. @simeonoff @zdrawku what do you think?

Closes #1501

### CODE REVIEW

**Comment by zdrawku** [comment]
@desig9stein sounds good.

File: en/components/themes/printing-styles.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Palettes
```
Comment (simeonoff) [inline en/components/themes/printing-styles.md]: This should probably be 'Print Layout' or something similar?

File: en/components/themes/printing-styles.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+_description: 
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, printing styles, @media print 
+---
+
+## Printing styles
+<p class="highlight">The Ignite UI for Angular theming engine provides some default printing styles, which make sure that our components have at leas the bare minimum to look decent when they are printed on a paper</p>
```
Comment (simeonoff) [inline en/components/themes/printing-styles.md]: Fix typos.

File: en/components/themes/printing-styles.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+<div class="divider--half"></div>
+
+### How to make use of the printing styles
+
+in order to make use of the printing styles, you need to make sure that your components are the same size as the selected document for print, otherwise, they will be cut off.
```
Comment (simeonoff) [inline en/components/themes/printing-styles.md]: Sentences should begin with a capital letter.

File: en/components/themes/printing-styles.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Printing styles
+<p class="highlight">The Ignite UI for Angular theming engine provides some default printing styles, which make sure that our components have at leas the bare minimum to look decent when they are printed on a paper</p>
+<div class="divider--half"></div>
+
+### How to make use of the printing styles
+
+in order to make use of the printing styles, you need to make sure that your components are the same size as the selected document for print, otherwise, they will be cut off. 
+
+For example, if you are planning to `print` a `Grid` in A4 format, you need to make sure that your `grid` size is exactly the same as the document for print. The `width and height` should match the size of the format that you want to prin in. Also, the data needs to fit in that view, without the need for scrollbars to appear.
```
Comment (simeonoff) [inline en/components/themes/printing-styles.md]: Fix typos:
prin -> print

**Comment by StefanIvanov** [comment]
@hanastasov can you review this PR it has been completed for a while now but has not been reviewed and merged yet.

**Comment by StefanIvanov** [comment]
Closed #1501

**Pull request merged**